docs/Commands.md

```diff
@@ -850,8 +850,7 @@ Gets completion suggestions for a prefix.
 - **prefix**: the prefix to complete on
 - **FUZZY**: if set, we do a fuzzy prefix search, including prefixes at Levenshtein distance of 1 from 
   the prefix sent
-- **MAX num**: If set, we limit the results to a maximum of `num`. (**Note**: The default is 5, and 
-  the number cannot be greater than 10).
+- **MAX num**: If set, we limit the results to a maximum of `num` (default: 5).
 - **WITHSCORES**: If set, we also return the score of each suggestion. this can be used to merge 
```

Comment:
This option is missing above in the format section

Reply:
Thanks - resolved via 2c60ae2